src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
+
+import { User } from '../../Types/user';
+
+type Props = {
```

Comment:
It is recommended to define the Props type using an interface instead of a type for better readability and consistency with other React components.